NodeJS/__integration__/AgentClient.it.ts

```diff
@@ -1,62 +1,81 @@
+      let result = await agentClient.waitForStationDeletion("agentNotExists")
+      expect(result).toBeTruthy()
+   })
+   test("waitForAgentDeletion should return true", async () => {
+      let result = await agentClient.waitForAgentDeletion("agentNotExists")
+      expect(result).toBeTruthy()
+   })
+   test("getSkillIds should return skillIds", async () => {
+      let skillIds = await agentClient.getSkillIds()
+      console.log(skillIds)
+      expect(skillIds.length).toBeGreaterThan(0)
+   })
+   test("getSkillNumbers should return skillNumbers", async () => {
+      let skillNumbers = await agentClient.getSkillNumbers()
+      expect(skillNumbers.length).toBeGreaterThan(0)
+   })
+   xtest("existsAgentByLoginId should return true when agent exists", async () => {
+      let exists = await agentClient.existsAgentByLoginId(7300000100)
+      expect(exists).toBeTruthy()
+   })
+   xtest("waitForAgentCreation should return true when agent exists", async () => {
+      let exists = await agentClient.waitForAgentCreation(7300000100)
+      expect(exists).toBeTruthy()
+   })
+   test.only("waitForDefaultSkillCreation should return true", async () => {
+      let exists = await agentClient.waitForDefaultSkillCreation()
+      expect(exists).toBeTruthy()
+      expect(agentClient.getDefaultSkillNumber()).toBeGreaterThan(0)
+   })
+   xtest("waitForAgentCreation should return false when agent does not exists", async () => {
+      let exists = await agentClient.waitForAgentCreation(3337300000100)
+      expect(exists).toBeFalsy()
+   })
+   xtest("existsAgentByUsername returns when agent exists by username", async () => {
+      let exists = await agentClient.existsAgentByUsername("ddksgy3dnr")
+      expect(exists).toBeTruthy()
+   })
+   xtest("existsStationForAgent should return false when agent has no station", async () => {
+      let exists = await agentClient.existsStationForAgent("ddksgy3dnr")
+      expect(exists).toBeFalsy()
+   })
+   xtest("getAgent should return agent and station", async () => {
+      let agentAndStation = await agentClient.getAgent("ddksgy3dnr")
+      expect(agentAndStation.agent).toBeDefined()
+      expect(agentAndStation.station).toEqual({})
+   })
+   xtest("createStationIfNotExists should return true", async () => {
+      let result = await agentClient.createStationIfNotExists("ddksgy3dnr", "2")
+      expect(result).toBeTruthy()
+   })
+   test("getUserToken should return token", async () => {
+      let result = await agentClient.getUserToken("yangadmin1")
+      expect(result).toEqual(token)
+   })
+   test("createDefaultSkill should return true.", async () => {
+      let result = await agentClient.createDefaultSkill()
+      expect(result).toBeTruthy()
+   })
+   test("getDefaultSkill should return skillId", async () => {
+      let result = await agentClient.fetchDefaultSkillNumber()
+      console.log(`skillNumber of default skill = ${result}`)
```

Comment:
remove `console.log`

Reply:
There are a whole bunch of `console.log` statements littered throughout this code. Why? I cannot imagine that these are useful long term, so I don't know why you would want them to continue to exist.

Reply:
I use integration test to see if code is correct.  It is meant to run manually and it lets me see what comes back from the server call.